### ISSUE TITLE
Images appear slightly blurred

### ISSUE THREAD

**Issue by ModiGrey** [Ruben2776/PicView#136]
**Describe the bug**
When viewing an image in picview 2.2 the image appears slightly blurred compared to other photo viewers I've tried. Below is an example of how the image should appear

**Expected behavior**
Images should appear sharper and less blurred

**Screenshots**
![Screenshot 2024-01-05 222123](https://github.com/Ruben2776/PicView/assets/140760417/cdf6ea9b-56d3-4152-a835-8ccfa2d7f080)

**Additional context**
Tried to change image aliasing to nearest neighbor, but it still the same.

**Comment by Ruben2776** [comment]
Do you have the setting `Stretch image` on?

**Comment by ModiGrey** [comment]
No, but I tried to turn it on to see if the issue is resolved, but nothing changed.

**Comment by Ruben2776** [comment]
@ModiGrey Sorry for the late reply, 
PicView version 3.0 and later has been rewritten to a new platform. Do you still experience this issue in the latest version?

**Comment by florient22** [comment]
I have the same problem

**Comment by Ruben2776** [comment]
@florient22 
Sorry for the slow response.  
In version 3.1.2 I have updated it to fix blurry images if you're using Window's scaling functionality (DPI). If you're not using scaling, then I'm unsure as to why you're getting the error though.

**Comment by barongi** [comment]
Same problem here. PicView 4.2.0 on Windows 11 25H2, scaling at 100%. Although not quite as blurry as the example here, but it softens the pictures a bit too much. There's a similar problem with the image viewer here https://github.com/AlienCowEatCake/ImageViewer. My ancient Pictus that I still use doesn't have this issue https://github.com/poppeman/Pictus
